app/react/kubernetes/ServicesView/types.ts

```diff
@@ -0,0 +1,34 @@
+type ServicePort = {
+  Name: string;
+  NodePort: number;
+  Port: number;
+  Protocol: string;
+  TargetPort: string;
+};
+
+type IngressStatus = {
+  Hostname: string;
+  IP: string;
+};
+
+type Application = {
+  UID: string;
+  Name: string;
+  Type: string;
+};
+
+export type Service = {
+  Name: string;
+  UID: string;
+  Namespace: string;
+  Annotations?: Record<string, string>;
+  Labels?: Record<string, string>;
+  Type: 'ClusterIP' | 'ExternalName' | 'NodePort' | 'LoadBalancer' | string;
```

Comment:
is the string required?

Reply:
It might not be.   I'm just wondering if I left it in and another type was introduced.